src/libecalc/dto/components.py

```diff
@@ -335,7 +319,7 @@ class Installation(BaseComponent):
     user_defined_category: Optional[InstallationUserDefinedCategoryType] = None
     hydrocarbon_export: Dict[datetime, Expression]
     fuel_consumers: List[Union[GeneratorSet, FuelConsumer, ConsumerSystem]] = Field(default_factory=list)
-    venting_emitters: List[VentingEmitter] = Field(default_factory=list)
+    venting_emitters: List[YamlVentingEmitter] = Field(default_factory=list)
```

Comment:
Here, it should be also the domain objcet `VentingEmitter`, and not the "yaml version". It may be that there is sth that I am missing, ie. that this is one step on the way, and a reason to keep it like this temporarly, but I dont see that looking at the code :)

Reply:
This also means that we need to send `variables_map` into `component_mapper`, so probably we will keep the `YamlVentingEmitter` here for now?

Reply:
As discussed, for now we keep the approach "from yaml to result".